Makefile

```diff
@@ -78,7 +78,27 @@ src/utils.o: src/utils.c src/utils.h
 
 src/ctr_logging.o: src/ctr_logging.c src/ctr_logging.h src/utils.h
 
-src/conmon.o: src/conmon.c src/cmsg.h src/config.h src/utils.h src/ctr_logging.h
+src/conmon.o: src/conmon.c src/cmsg.h src/config.h src/utils.h src/ctr_logging.h src/globals.h src/cli.h src/oom.h src/cgroup.h src/conn_sock.h src/ctrl.h src/ctr_stdio.h
```

Comment:
I am woefully not knowledgeable about Makefiles. I am sure there's a better way to do this, but I don't know about it.